db/migrate/20201015161629_change_type_company_type.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTypeCompanyType < ActiveRecord::Migration[6.0]
+  def change
+    change_column :companies, :company_type, 'integer USING CAST(company_type AS integer)'
```

Comment:
can you explain what  do`'integer USING CAST(company_type AS integer)'`?

Reply:
change type company_type from string to integer

Reply:
> change type company_type from string to integer

![image](https://user-images.githubusercontent.com/33605808/96242015-a6f79600-0fab-11eb-9171-19e4b15735b0.png)